tmt/steps/execute/internal.py

```diff
@@ -91,6 +91,8 @@ def log(key, value=None, color=None, shift=1, level=1):
             stdout = error.stdout
             test.returncode = error.returncode
         end = time.time()
+        if type(stdout) is tuple or type(stdout) is list:
+            stdout = ' '.join(stdout)
```

Comment:
It was mentioned [here](https://github.com/psss/tmt/pull/294#issuecomment-655812378); it also should be reproducable.

The cause of this was that some plan inheriting from `main` plan has `list`, and I've used `string` or vice versa; which rendered the inheritance incompatible. (I don't remember precisely.)

I think either correct error message or this fix is needed.

Reply:
Weird. Here `stdout` should never be tuple or list because 'join=True' is used `guest.execute()` above. If you are able to reproduce on master, let's solve this as a separate issue.